### DISCUSSION TITLE
[Feature Suggestion] Fire Extinguisher Recycle Recipe

### DISCUSSION THREAD

**Discussion by Next-Door-Tech** [FakeFishGames/Barotrauma#12615] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Short and simple suggestion - a recipe to recycle fire extinguishers. Should require an empty extinguisher (<10% durability) and two sodium (instead of one aluminum and two sodium) in keeping with the rest of the existing recycle recipes.

On another note, is GitHub the preferred place to submit feature suggestions like this or is somewhere else better? (Wondering because you don't have a feature suggestion issue template.)

### Reproduction steps

Not Applicable

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.0.21.0

### Which operating system did you encounter this bug on?

Windows

**Comment by jigardi** [comment]
I like this idea have my upvote!
I also turned this into an override mod here: https://steamcommunity.com/sharedfiles/filedetails/?source=Facepunch.Steamworks&id=3443581990
